### ISSUE TITLE
Add `eslintrc` configuration for wxt's auto-import to avoid eslint errors

### ISSUE THREAD

**Issue by chillcicada** [wxt-dev/wxt#448]
### Feature Request

In the vite + unplugin-auto-import project, i use the eslintrc parameter for AutoImport to generate the corresponding json file for eslint to import and avoid eslint errors. I hope that wxt can introduce related functionality in the future to avoid reinstalling unplugin-auto-import and reconfiguring vite internally in wxt project.

#### Is your feature request related to a bug?

N/A

### What are the alternatives?

Similar to unplugin-auto-import/vite, add the `eslintrc` configuration under the imports parameter, following the syntax of unplugin-auto-import/vite. This will generate the json file while generating the types. The config code will seem to look like this:

```ts
// wxt.config.ts
export default defineConfig({
  imports: {
    ...
    eslintrc: {
      enabled: true,
      filepath: './.eslintrc-auto-import.json',
      globalsPropValue: true,
    },
  },
  ...
})
```

**Comment by aklinker1** [comment]
Seems easy enough: https://github.com/unplugin/unplugin-auto-import/blob/main/src/core/eslintrc.ts

**Comment by aklinker1** [comment]
Released in [`v0.16.7`](https://github.com/wxt-dev/wxt/releases/tag/v0.16.7). See the PR https://github.com/wxt-dev/wxt/pull/450 for usage.
